listenbrainz/background/listens_importer/base.py

```diff
@@ -22,6 +22,20 @@
 IMPORTER_NAME = "ListenBrainz Archive Importer"
 
 
+def update_import_task(db_conn, import_id: int, **kwargs):
+    """Update import task status and progress."""
+    query = text("""
+                 UPDATE user_data_import
+                 SET metadata = metadata || (:metadata)::jsonb
+                 WHERE id = :import_id
+                 """)
+    db_conn.execute(query, {
+        "metadata": json.dumps(kwargs),
```

Comment:
Just for my own curiosity, can this throw an error if we somehow send wrong kinds of arguments?